docs/guides/setup/dpdk-setup-guide.md

```diff
@@ -17,6 +17,9 @@ For build instructions, see [P4 SDE Readme](https://github.com/p4lang/p4-dpdk-ta
 
 ### Install basic utilities
 
+Note: See [OpenSSL security guide](/guides/security/openssl-guide.md)
```

Comment:
You have a heading to call attention to the following line. There's no need to add "Note:".

```suggestion
See the [OpenSSL security guide](/guides/security/openssl-guide.md)
```